@l10n/ja/docs/concepts/tokens/fungible-tokens/deep-freeze.md

```diff
@@ -0,0 +1,92 @@
+---
+blurb: ディープフリーズは、トラストラインの凍結が解除されるまで、凍結されたトークン保有者がその資金を送受信できないようにします。
+labels:
+  - トークン
+  - 凍結
+  - ディープフリーズ
+---
+# ディープフリーズ
+
+ディープフリーズは、XRP Ledger上のトークン発行者が、凍結されたアカウントによるトークンの悪用を防ぐための機能です。これは、凍結された資産と支払いのやり取りを強化し、トラストラインが凍結解除されない限り、凍結されたトークン保有者が資金を受け取ることができないようにします。
+
+この変更により、発行者はXRPL上での規制遵守がより容易になります。たとえば、制裁リストに記載されたウォレットへのトークン流通を防ぐことで、規制されたステーブルコインや現実資産（RWA）などのユースケースにおいて、コンプライアンスを強化できます。
+
+ディープフリーズは、支払い、オファー、分散型取引所（DEX）、自動マーケットメイカー（AMM）に関わるトラストラインに影響を与えます。発行者は、ディープフリーズ状態の保有者に対して、送金および受取をブロックできます。
+
+_([DeepFreeze amendment][]により追加されました。)_
+
+## ディープフリーズの仕組み
+
+ディープフリーズは、トラストライン上に設定される機能です。これを有効にするには、発行者がまず通常の凍結（Individual Freeze）をそのトラストラインに対して実行しておく必要があります。また、発行者のアカウントで `No Freeze` が有効になっている場合は、ディープフリーズを実行することはできません。
+
+発行者がディープフリーズを実行すると、そのトラストライン上のトークンには以下のルールが適用されます。
+
+- ディープフリーズされたトラストラインの両当事者間の直接決済は、ディープフリーズ後も可能です。
+- トラストラインのカウンターパーティは、発行者への直接決済を除き、ディープフリーズされたトラストライン上の残高を増減させることができません。カウンターパーティは、ディープフリーズされた通貨を発行者にのみ直接送ることができます。
+- カウンターパーティは、ディープフリーズされたトラストライン上で、他のアドレスとの間で決済を行うことができません。
+- カウンターパーティによる、ディープフリーズされたトラストライン上のトークンの売買オファーは資金不足と見なされます。
+
+また、個別のアドレスが、自分から発行者または金融機関へのトラストラインをディープフリーズに設定することも可能です。
+これは、その金融機関と他のユーザー間の取引には影響を与えませんが、次の効果があります。
+
+- 他のアドレスが、その金融機関のトークンをその個別アドレスに送信することを防ぎます。
+- その個別アドレスが、そのトークンを発行者以外のアドレスに送信することも防ぎます。
+
+### RippleState オブジェクト
+
+ディープフリーズでは、`RippleState`（トラストライン）オブジェクトに、`lsfLowDeepFreeze` と `lsfHighDeepFreeze` の2つのフラグが導入されます。
+
+| フラグ名	| フラグ値	| 説明 |
+|-----------|-------------|-------------|
+| `lsfLowDeepFreeze`	| `0x02000000`	| 低位アカウントがトラストラインをディープフリーズしており、高位アカウントは資産を移動できません。 |
+| `lsfHighDeepFreeze`	| `0x04000000`	| 高位アカウントがトラストラインをディープフリーズしており、低位アカウントは資産を移動できません。 |
+
+[高位vs低位アカウント](../../../references/protocol/ledger-data/ledger-entry-types/ripplestate.md#high-vs-low-account)をご覧ください.
+
+### TrustSet トランザクション
+
+ディープフリーズでは、`TrustSet` トランザクションに `tfSetDeepFreeze` および `tfClearDeepFreeze` の2つのフラグが導入されます。
+
+| フラグ名	| フラグ値	| 説明 |
+|-----------|-------------|-------------|
+| `tfSetDeepFreeze`	| `0x00400000`	| トラストラインをディープフリーズします。 |
+| `tfClearDeepFreeze`	| `0x00800000` | トラストラインのディープフリーズを解除します。 |
+
+`tfSetDeepFreeze` を設定しようとする `TrustSet` トランザクションは、以下のいずれかが当てはまる場合にのみ成功します。
+
+- 保有者はすでに凍結されており、その状態はトラストライン上の `lsfLowFreeze`/`lsfHighFreeze` フラグによって示されている。
+- 同じ `TrustSet` トランザクションで `tfSetFreeze` も同時に設定されている。
+
+ディープフリーズの導入により、`TrustSet` トランザクションには追加の制限が課されます。
+
+トラストラインが発行者によってディープフリーズされている場合（`lsfLowDeepFreeze`/`lsfHighDeepFreeze` フラグが立っている）、発行者が `tfClearDeepFreeze` フラグを設定せずに `tfClearFreeze` フラグだけを設定すると、`TrustSet` トランザクションは失敗します。つまり、発行者はディープフリーズを解除しないかぎり、通常の凍結を解除することはできません。
+
+## 支払いエンジン
+
+支払いエンジンは、送信者と受信者をつなぐステップで構成されたパスを実行します。一般的に、資金は次の二つのうちいずれかのステップを通じてトラストラインに入金されます。
+
+- [リップリング](#rippling)
+- [オーダーブックまたはAMM](#order-book)
+
+### リップリング
+
+リップリングのステップの結果として、ディープフリーズされたトラストラインでの資金受け取りは失敗します。
+
+### オーダーブック
+
+買い注文で指定された `TakerPays` トークンについて、オファー所有者のトラストラインがディープフリーズされていると、そのオファーは資金の裏付けがないと判断され、成立しません。
```

Comment:
Pick is up to you, but on other pages, we use “約定” instead of “成立”

```suggestion
買い注文で指定された `TakerPays` トークンについて、オファー所有者のトラストラインがディープフリーズされていると、そのオファーは資金の裏付けがないと判断され、約定しません。
```